packaging/release/Makefile

```diff
@@ -0,0 +1,60 @@
+version ?= $(shell python versionhelper/version_helper.py --raw)
+
+.PHONY: all
+all:
+	@echo "USAGE:"
+	@echo
+	@echo "make release version={version} # current version is '${version}'"
+	@echo "make publish"
+	@echo
+	@echo "NOTE: Make sure to source hacking/env-setup before running these targets."
```

Comment:
How bad is it if someone doesn't do this?

Do we need to detect/fail gracefully if this hasn't been done, or as we assuming that people are following the documented process so this should be OK?

Reply:
I think it's also important to note that this isn't a new issue. Our current manual instructions also require sourcing of `hacking/env-setup`, with the same consequences if you don't.